package.json

```diff
@@ -27,37 +26,38 @@
   "author": "Department for Environment, Food & Rural Affairs",
   "license": "See LICENSE.md",
   "engines": {
-    "node": "^8.12.0",
-    "npm": "^6.4.1"
+    "node": "^12.13.1",
+    "npm": "^6.12.1"
   },
   "dependencies": {
+    "@hapi/cookie": "^11.0.0",
+    "@hapi/hoek": "^9.0.3",
+    "@hapi/joi": "^15.1.1",
```

Comment:
I would recommend you adopt the same versioning strategy as Hapi. (https://github.com/hapijs/hapi/blob/master/package.json)
This will allow for dependencies to be auto upgraded, for example define joi as:
```
    "@hapi/joi": "17.x.x",
```

Reply:
Hey Pedro,

We have very stringent rules around automatically upgrading things, especially when they are to be used by other services so unfortunately we can't do that at the moment, but it's something we will consider for the future.